shell_integration/windows/OCContextMenu/OCClientInterface.cpp

```diff
@@ -72,7 +72,8 @@ std::shared_ptr<HBITMAP> saveImage(const string &data)
     Gdiplus::GdiplusStartupInput gdiplusStartupInput;
     Gdiplus::GdiplusStartup(&gdiplusToken, &gdiplusStartupInput, nullptr);
 
-    DWORD size = 2 * 1024;
+    const int bytesPerKibibyte = 1024;
+    DWORD size = 2 * bytesPerKibibyte;
```

Comment:
maybe lift my `_kb` operator from `testchunkingng.cpp:19`, put it in a utility file, and use it here?